graphics/graphics_fonts.xml

```diff
@@ -163,6 +163,14 @@
 		<ttf file="graphics/fonts/Roboto-Bold.ttf" name="Roboto Bold" size="16" />
 		<color value="#DDDDDD" /> <!-- Replaceable Color: Secondary Foreground -->
 	</font>
+	<font name="npcsheet_header">
+		<ttf file="graphics/fonts/Noto_Sans/NotoSans-Bold.ttf" name="Noto Sans Bold" size="16" />
+		<color value="#FFF0CA" />
+	</font>
+	<font name="storybox_header">
+		<ttf file="graphics/fonts/Noto_Sans/NotoSans-Bold.ttf" name="Noto Sans Bold" size="24" />
+		<color value="#FFF0CA" />
+	</font>
```

Comment:
These two entries are for the font visibility fix.  Everything else in the file is from the .21 Knightowl changes.